lisp/ess-custom.el

```diff
@@ -191,7 +191,9 @@ lot."
   "When non-nil, set the width option when the window configuration changes.
 When 'frame, set the width to the frame width. When 'window, set
 the width to the window width. If an integer, set the width to
-that integer. Anything else is treated as 'window."
+that integer. If it's a negative integer, set the width to the
```

Comment:
Looks like in the code it's the positive integer and which is negated at the end.

Reply:
I don't think so?

```
(ess-calculate-width 'window)
"options(width=158, length=99999)
"

(ess-calculate-width -2)
"options(width=156, length=99999)
"

(ess-calculate-width 2)
"options(width=2, length=99999)
"
```